ios/FluentUI/DrawerVnext/Drawer+UIKit.swift

```diff
@@ -77,15 +79,19 @@ open class DrawerVnext: UIHostingController<AnyView>, FluentUIWindowProvider {
         return self.view.window
     }
 
+    /// Set this delegate to recieve updates when drawer's state changes
+    /// @see `DrawerVnextControllerDelegate`
     public weak var delegate: DrawerVnextControllerDelegate?
 
+    /// Represents the drawer's state with properties to configure its behavior.
+    /// @see `DrawerState`
     @objc open var state: DrawerState {
         return self.drawer.state
     }
 
     @objc public init(contentViewController: UIViewController,
                       theme: FluentUIStyle? = nil) {
-        let drawer = Drawer(content: DrawerContentViewController(contentViewController: contentViewController))
+        let drawer = Drawer(content: SwiftUIAdapter.UIViewControllerAdapter(contentViewController))
```

Comment:
Lots of "Adapter" in this call. How about removing the namespace and have the swift file called UIKitAdapter**s**.swift (plural)? (It's probably better than SwiftUIAdapters since we have adapters for UIKit types)

Reply:
I assume the intent of using a separate file is to create a module. File with orphan class or loosely based modules usually encourage generic programming or doesn't provide clear directions for extensions.

Naming convention could be adjusted We can have any of the following

- `SwiftUIAdapter.UIView` 
-  'SwiftUIAdapter.UIViewController` 
-  'UIKitAdapter.UIViewAdapter` 
-  'UIKitAdapter.UIViewControllerAdapter` 

I like something that clearly articulates for a UIKit client that this is to be used when integrating with SwiftUI only. 
I am open to any of the above options, thoughts??

Reply:
The idea behind moving the code to a separate file is organizing the code.

One of the benefits of organizing them like this is: when we're publishing subspecs on cocoapods we are preventing another control that uses these adapters from depending on the Drawer or List.

Since these files contain code that can be used by multiple controls, it's a good idea to move them to their own file.

Maybe the file could be named **UIKit+SwiftUI_interoperability.swift** and we have the times named as they are with no namespace? (**UIViewAdapter** and **UIViewControllerAdapter**)?